docs/azmcp-commands.md

```diff
@@ -449,7 +449,42 @@ azmcp monitor workspace log query --subscription <subscription> \
                                   --table-name "AppEvents_CL" \
                                   --query "| order by TimeGenerated desc"
 ```
-
+### Azure Arc Operations
+```bash
+# Validate system requirements and setup Hyper-V for AKS Edge Essentials
+azmcp arc validate-system-requirements-hyperv --path <path>
+
+# Validate prerequisites for AKS Edge Essentials cluster
+azmcp arc validate-prerequisites-aksee-cluster
+
+# Validate and install software requirements for AKS Edge Essentials
+azmcp arc validate-install-software-requirements --path <path>
+
+# Completely remove AKS Edge Essentials from the machine
+azmcp arc remove-Aks-Edge-installation --path <path>
```

Comment:
Why is A and E captial in Aks ? Edge ?

should follow aks-edge same as other commands.